plugins/systemd/timesync_status

```diff
@@ -87,11 +90,14 @@ def retrieve():
     output = result.stdout.decode('utf-8')
     values = parse_response(output)
 
-    print('offset.value', parse_time(values['Offset']))
-    print('delay.value', parse_time(values['Delay']))
-    print('delay.extinfo', 'Server', values['Server'])
-    print('jitter.value', parse_time(values['Jitter']))
-    print('poll.value', parse_time(values['Poll interval'].split('(')[0]))
+    if 'Offset' in values:
+        print('offset.value', parse_time(values['Offset']))
+        print('delay.value', parse_time(values['Delay']))
+        print('delay.extinfo', 'Server', values['Server'])
+        print('jitter.value', parse_time(values['Jitter']))
+        print('poll.value', parse_time(values['Poll interval'].split('(')[0]))
+    else:
+        print('delay.value -1')
```

Comment:
Is `-1` a suitable value here? How about using `U` (*unknown*) for *all* fields instead?

(a quick comment, which condition leads to this branch would be perfect!)

Reply:
right, working on it